x-pack/plugins/console_extensions/spec/overrides/ilm.put_lifecycle.json

```diff
@@ -0,0 +1,54 @@
+{
+  "ilm.put_lifecycle": {
+    "data_autocomplete_rules": {
+      "phases": {
+        "warm": {
```

Comment:
it would be great if we can add `_templates` here for the most basic config, to give people an idea of the structure.

Reply:
done